homeassistant/components/samsungtv/const.py

```diff
@@ -9,3 +9,11 @@
 CONF_MANUFACTURER = "manufacturer"
 CONF_MODEL = "model"
 CONF_ON_ACTION = "turn_on_action"
+CONF_TOKEN_FILE = "token_file"
+
+PORTS = (55000, 8001, 8002)
```

Comment:
Not needed anymore.

Reply:
I'll add` CONF_TOKEN_FILE` with `CONF_TOKEN` (and delete the `PORTS`)